### PR TITLE
RED-1848: update aviary.yaml with proper excludes

### PR DESCRIPTION
**Pull request by jeffreymelvin-wf**
@stevenosborne-wf @brianshannan-wf 
@infosec-wf @matthewlorimor-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 1</summary>
<li>Watched file aviary.yaml modified</li>
</details>

**Comment by matthewlorimor-wf** [comment]
Security +1

File: aviary.yaml [hunk 1]
```diff
@@ -2,12 +2,7 @@ version: 1
...
   - tests?/
-
-raven_monitored_classes: null
-
-raven_monitored_files: null
-
-raven_monitored_functions: null
-
-raven_monitored_keywords: null
+  - godeps/
```
Comment (brianshannan-wf) [inline aviary.yaml]: Is aviary case insensitive? This directory should be "Godeps"

Comment (matthewlorimor-wf) [reply aviary.yaml]: It's case insensitive.

**Comment by brianshannan-wf** [comment]
+1

**Comment by charliestrawn-wf** [comment]
+1

**Pull request merged**